### DISCUSSION TITLE
How to get full access token from Supabase dashboard for setting up Github actions

### DISCUSSION THREAD

**Discussion by ngamsomset** [supabase/supabase#21475] [Questions] [answered]
I'm trying to setup Github actions according to [this](https://supabase.com/docs/guides/cli/managing-environments?environment=staging#create-a-new-migration). There is one secret that I need to put in Github secrets which is access token. I can see them from the Supabase dashboard but it does contains *** in the middle, I assume that's for security reason. Is there a way I can get full token to put into Github secrets?

**Comment by silentworks** [comment, marked as answer]
The ones with *** in the middle are tokens you've generated before, if you did not save them somewhere you cannot view them again however you can just generate new ones which will show you the full token right after you generate it and then use that. I don't think there is a limit on the number of tokens you can generate so I would generate a new one.
